doc/howto/index.rst

```diff
@@ -0,0 +1,20 @@
+======
+Howtos
+======
+
+.. toctree::
+   :maxdepth: 1
+
+   api_parameters
```

Comment:
It seems you've sorted the howto docs alphabetically by name, but thematically would be a better order, IMHO.

Specifically, the first entry, `api_parameters` would logically belong in a group with the last entry, `using_the_api` (it is also referenced from there).  However, looking at `api_parameters`, I would argue this document doesn't even belong in the howto section to begin with - it's reference material!

Two suggestions:
Either fix in this PR by moving `api_parameters` to the reference section, or just group it with `using_the_api` and fix the section-moving in a separate PR (either version works for me, this isn't code and doesn't need to be entirely logical :D )

Reply:
It started with `ls -1 | grep ".rst" > index.rst` so no, no deliberate ordering whatsoever :)